lbz/lambdas/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .broker import LambdaBroker
+from .enums import LambdaResult, LambdaSources
+from .invoker import LambdaClient
```

Comment:
I would name this module simply `client.py` if you already put `LambdaClient` inside 😉